web/app/themes/justicejobs/src/js/job-search-form.js

```diff
@@ -3,23 +3,6 @@ The Job Form Search Functionality
 */
 jQuery(document).ready(function ($) {
 
-
-    var resultsViewWrapper = $('.search_contain__container');
-    var listButton = $('.search_contain__label--list');
-    var mapButton = $('.search_contain__label--map');
-
-    listButton.on('click', function () {
-        resultsViewWrapper.removeAttr('id', 'js-show-map').attr('id', 'js-hide-map');
-        mapButton.attr('aria-pressed', 'false');
-        listButton.attr('aria-pressed', 'true');
-    });
-
-    mapButton.on('click', function () {
-        resultsViewWrapper.attr('id', 'js-show-map');
-        listButton.attr('aria-pressed', 'false');
-        mapButton.attr('aria-pressed', 'true');
-    });
-
```

Comment:
This code was removed, relocated and relocated to `maps.js` so I could take advantage of the local map redraw functionality when the buttons were clicked.